src/OrbitQt/HistogramWidget.cpp

```diff
@@ -320,4 +310,22 @@ HistogramWidget::GetSelectionRange() const {
 
 void HistogramWidget::EmitSignalSelectionRangeChange() const {
   emit SignalSelectionRangeChange(GetSelectionRange());
-}
\ No newline at end of file
+}
+
+constexpr size_t kMaxFunctionNameLengthForTitle = 80;
+
+[[nodiscard]] QString HistogramWidget::Title() const {
```

Comment:
As per the style guide, this function should either be called `title` (to imitiate a public member) or `GetTitle`. I think the latter is more appropriate in this case.